changehc/tests/test_classconfig.py

```diff
@@ -0,0 +1,13 @@
+# standard
+import pytest
+
+# first party
+from delphi_emr_hosp.config import Config
```

Comment:
are the tests just copy pasted for now? I see a lot of references to emr_hosp objects

Reply:
Yes, just copied and pasted. I wasn't expecting this to go to code review yet. I'm currently working on the tests.

Reply:
:+1: cool, just ping me when it's ready to take a look at

Reply:
@chinandrew I've added tests, it's ready for you to take a look